android/src/main/java/com/reactnativepagerview/PagerViewViewManagerImpl.kt

```diff
@@ -7,6 +7,8 @@ import com.facebook.react.uimanager.PixelUtil
 object PagerViewViewManagerImpl {
     const val NAME = "RNCViewPager"
 
+    var animated = true
```

Comment:
Do we need to set this?  

Since the prop is defining it's default value here: `@ReactProp(name = "animated", defaultBoolean = true)`

Reply:
It has to be set, because `lateinit` is supported for properties of object types only, not primitive types